src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -1,127 +1,148 @@
 <template id="limited-restoration-panel">
-  <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
-    <v-radio class="radio-button pt-2" id="twentyFour-radio" name="selectMonths" label="2 years" :value=24 />
-    <v-radio class="radio-button pt-2" id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
-    <v-radio class="radio-button pt-2" id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
-    <v-radio class="radio-button pt-2" id="six-radio" name="selectMonths" label="6 months" :value=6 />
+  <v-radio-group
+    v-model="radioValue"
+    class="mt-0 pt-0 ml-8"
+    mandatory
+  >
+    <v-radio
+      id="radio-24"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="2 years"
+      value="24"
+    />
+    <v-radio
+      id="radio-18"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="18 months"
+      value="18"
+    />
+    <v-radio
+      id="radio-12"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="12 months"
+      value="12"
+    />
+    <v-radio
+      id="radio-6"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="6 months"
+      value="6"
+    />
     <v-row class="ml-0 mt-0 radio-button pt-2">
-      <v-radio class="mt-n4" id="custom-months" name="selectMonths" value="customMonths" />
+      <v-radio
+        id="radio-custom"
+        class="mt-n4"
+        name="radioValue"
+        value="customMonths"
+      />
       <v-form ref="monthsRef">
         <v-text-field
-          id="months-text-field"
-          class="number-months-field"
+          id="text-field-months"
+          v-model="inputValue"
+          class="text-field-months"
           type="number"
           dense
+          filled
           hide-spin-buttons
-          :rules="monthRules"
-          v-model="numberOfMonths"
-          :disabled="!customMonths"
-          @change="onMonthsChanged"
-          filled />
+          :rules="monthsRules"
+          :disabled="(radioValue !== 'customMonths')"
+          @input="onMonthsInput($event)"
+        />
       </v-form>
-      <div class="ml-2 mt-2 month-text">month(s)</div>
+      <span class="ml-2 mt-2 month-text">month(s)</span>
     </v-row>
   </v-radio-group>
 </template>
 
 <script lang="ts">
-import { Component, Emit, Mixins, Prop, Watch } from 'vue-property-decorator'
-import { DateMixin } from '@/mixins' // NB: local mixin (StoryBook can't find it otherwise)
+import Vue from 'vue'
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
 import { FormIF } from '@bcrs-shared-components/interfaces'
 
 @Component({})
-export default class LimitedRestorationPanel extends Mixins(DateMixin) {
+export default class LimitedRestorationPanel extends Vue {
```

Comment:
Why did you change this back to extend Vue Sev? In the UIs, we were moving towards extending Mixins. Why is it not the case here?

Reply:
This component no longer needs the mixin :)

Reply:
Oh OK. My bad. Resolving!

Reply:
No, I'm glad you're looking for such things and questioning them 😃